docs/src/app/components/pages/components/paper.jsx

```diff
@@ -64,7 +64,7 @@ const PaperPage = React.createClass ({
           },
           {
             name: 'zDepth',
-            type: 'number (0-5)',
+            type: 'oneOf ["0", "1", "2", "3", "4", "5"]',
```

Comment:
I think you should use the value exactly as one would actually have to type in code. So for instance, for `number` `PropTypes`, I wouldn't use quotes:

`<Paper zDepth = {1}/>`

but for `string` I would.